recipes/influxdb-python/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "influxdb-python" %}
+{% set version = "5.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/i/influxdb/influxdb-{{ version }}.tar.gz
+  sha256: afeff28953a91b4ea1aebf9b5b8258a4488d0e49e2471db15ea43fd2c8533143
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - python-dateutil >=2.6.0
+    - pytz
+    - requests >=2.17.0
+    - six >=1.10.0
+  run:
+    - python
+    - python-dateutil >=2.6.0
+    - pytz
+    - requests >=2.17.0
+    - six >=1.10.0
+
+test:
+  imports:
+    - influxdb
+    - influxdb.influxdb08
+    - influxdb.tests
+    - influxdb.tests.influxdb08
+    - influxdb.tests.server_tests
+  requires:
+    - mock
+    - nose
+    - nose-cov
+    - requests-mock
+
+about:
+  home: https://github.com/influxdb/influxdb-python
+  license: MIT
+  license_family: MIT
+  license_file: 
```

There should be a licence file, if it's not included in the tarball you can also put it in the recipe directory.
```suggestion
  license_file: LICENSE
```